recipes/metamdbg/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.3" %}
+
+package:
+  name: metamdbg
+  version: {{ version }}
+
+source:
+  url: https://github.com/GaetanBenoitDev/metaMDBG/archive/refs/tags/metaMDBG-{{ version }}.tar.gz
+  sha256: 24b34fcc7416b44081963a470059209c7f0592bd597a25388a94dae5edb2b09d
+
+build:
+  skip: True  # [osx]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('metamdbg', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - zlib
+    - openmp
+  run:
+    - zlib
+    - openmp
+    - wfmash =0.8.2
+    - samtools =1.6
+    - minimap2 =2.24
+    - gsl =2.7 =he838d99_0
```

Comment:
Are such strict dependency pinnings absolutely necessary here?

Reply:
For gsl and wfmash: it is a fix to have wfmash working (the fix is described here https://github.com/pangenome/pggb/issues/196), the latest wfmash version on bioconda is not working, I don't know why it has never been fixed.

For other packages, it is to ensure reproducibility.

Reply:
I see, thanks for the explanation!